### ISSUE TITLE
[WinBuild] VS2013 build errors after #2420

### ISSUE THREAD

**Issue by RichardHaselgrove** [BOINC/boinc#2483]
#2420 (use per-thread locale on Linux) introduces explicit

`#include "config.h"`

directives to clientgui/AsyncRPC.cpp and lib/gui_rpc_client.h, for all platforms. VS2013 appears not to use config.h: there is a null (empty) file in boinc\win_build, but VS2013 didn't find it and errored the build with 'unable to open config.h' on those two files.

Workround - placing an additional copy of the dummy (empty) file in each of the two folders allowed the build to complete and run normally, but it would be neater if this could be fixed via the solution or dependency files.

**Comment by AenBleidd** [comment]
@RichardHaselgrove, according to this log everything is ok: https://ci.appveyor.com/project/BOINC/boinc/branch/master

**Comment by RichardHaselgrove** [comment]
Yes, appveyor is obviously configured to work round or eliminate the problem I encountered.

But on my standalone Windows 7 machine, configured according to https://boinc.berkeley.edu/trac/wiki/CompileClient#Windows, I encountered the errors I described. But I was able to work round them and verify that there was nothing fundamentally wrong with the code - just the build environment.

**Comment by JuhaSointusalo** [comment]
@RichardHaselgrove

You need to include #2311 to the client_release, not because fixing the simulator is interesting for releasing a client but because it includes project file updates and #2420 needs those updates.

**Comment by RichardHaselgrove** [comment]
@JuhaSointusalo - ah, thank you. Good thing I mentioned it!

@TheAspens, @davidpanderson - could one of you cherrypick #2311 across to the 7.10 branch, please?

**Comment by RichardHaselgrove** [comment]
@JuhaSointusalo - I requested a pause on the release build so that this problem could be cleaned up, but we seem to be very short-staffed on reviewers and cherrypickers this week. Looking through all the discussion and code changes, it all seems to be about the Windows build process - which explains my problems with VS2013, and might cause problems for David with his VS2010 builds, but I don't see how it might impact on the Linux problems in #2420.

Having said that, we have a report - edit, two reports - of the client not running, or manager-client communications failing (which can be two sides of the same coin) for v7.10.1 under Mint 18.3 and Ubuntu 17.10. They're in http://boinc.berkeley.edu/dev/forum_thread.php?id=12425

Not a lot of detail to go on, but could you have a look to see if they could be caused by this issue, or whether we have to look elsewhere?
